src/pages/[platform]/index.tsx

```diff
@@ -69,7 +80,42 @@ const PlatformOverview = ({ platform }) => {
           other resources will help you build, connect, and host fullstack apps
           on AWS. Get started by selecting your preferred framework.
         </Text>
+
         <FrameworkGrid currentKey={platform} />
+        <LinkCardCollection>
+          {linkCardPlatformData ? (
+            <LinkCard
+              isExternal={true}
+              href={''}
```

Comment:
Does the `href` value also need to be updated here with the variable we want to display?

Reply:
done